### PR TITLE
LineaStatebridge

### PR DESCRIPTION
**Pull request by Jemiiah**
issue: https://github.com/kfastov/worldcoin-bridge-linea/issues/4
creating a comprehensive end-to-end test for the entire World ID state bridge system. to ensure that all components work together correctly and that the cross-chain functionality operates as expected.

### CODE REVIEW

**Comment by Jemiiah** [comment]
still working on a few things

**Comment by kfastov** [comment]
Hey @Jemiiah ! Sorry for ruining your previous PR, I just thought you abandoned it and I wanted to merge the tests quickly and failed. Thanks for working on it!

**Comment by Jemiiah** [comment]
thank you @kfastov would try and round things up

**Comment by Jemiiah** [comment]
@kfastov so what is left is replicating the linearworldID test which would be sorted out soonest ser sorry on the delay would sort that out soonest

File: remappings.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
-world-id-state-bridge=node_modules/@worldcoin/world-id-state-bridge/src
-linea-contracts/=node_modules/linea-contracts/contracts
+@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/
+forge-std/=node_modules/forge-std/src/
+world-id-state-bridge/=node_modules/@worldcoin/world-id-state-bridge/src/
+linea-contracts/=node_modules/linea-contracts/contracts/
+@prb/test/=node_modules/@prb/test/src/
+@eth-optimism/=node_modules/@eth-optimism/
+@worldcoin/=node_modules/@worldcoin/
+hardhat/=node_modules/hardhat/
```
Comment (kfastov) [inline remappings.txt:9]: Do we really need Hardhat btw? I haven't found any Hardhat or HRE mentions in your tests yet.

**Comment by kfastov** [comment]
@Jemiiah Sounds great. Looking forward to it 👍

File: remappings.txt (hunk 1 above)
Comment (Jemiiah) [reply remappings.txt]: by bad took care of it @kfastov

**Comment by Jemiiah** [comment]
@kfastov you can take a look now ser

**Comment by kfastov** [comment]
Hi @Jemiiah can you fix lint and build please?

**Pull request closed without merging**